agent/xds/jwt_authn.go

```diff
@@ -16,6 +16,16 @@ import (
 	"google.golang.org/protobuf/types/known/wrapperspb"
 )
 
+var (
+	jwtEnvoyFilter = "envoy.filters.http.jwt_authn"
+	jwtPayload     = "jwt_payload"
+)
+
+type JWTAuthnProvider struct {
```

Comment:
nit: We can make this unexported (private). 

The type/fields would benefit from a comment ("this is a helper that associates the provider with a unique name", etc)